pkg/tfbridge/schema_test.go

```diff
@@ -2588,6 +2589,200 @@ func TestExtractDefaultIntegerInputs(t *testing.T) {
 	assert.Equal(t, expected, ins)
 }
 
+func TestExtractSchemaInputsNestedMaxItemsOne(t *testing.T) {
+
+	provider := func(info *ResourceInfo) *Provider {
+		if info == nil {
+			info = &ResourceInfo{}
+		}
+		if info.Tok == "" {
+			info.Tok = tokens.NewTypeToken("module", "importableResource")
+		}
+
+		listOfObj := func(maxItems int) *schemav1.Schema {
```

Comment:
prefer schemav2 for new tests perhaps? or is cloudflare on schemav1

Reply:
Cloudflare is on v2. This test doesn't really care about the underlying provider framework, but I can switch to v2.